models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -233,14 +215,29 @@ aeif_cond_beta_multisynapse::Parameters_::set( const DictionaryDatum& d )
 
   updateValue< double >( d, names::gsl_error_tol, gsl_error_tol );
 
-  if ( V_peak_ <= V_th )
+  if ( V_reset_ >= V_peak_ )
```

Comment:
Ok, then this is probably my fault ^^"
No, just change it here, I'll correct the ``aeif_cond_alpha`` it in the PR suppressing the ``dynamics_DT0``